extensions/commands/recipe/README.md

```diff
@@ -36,3 +36,33 @@ updating moltenvk/1.2.0 to moltenvk/1.2.2 in conan-center-index/recipes/qt/6.x.x
 updating libjpeg-turbo/2.1.4 to libjpeg-turbo/2.1.5 in conan-center-index/recipes/qt/6.x.x/conanfile.py:386
 Successfully bumped the dependencies of recipe conan-center-index/recipes/qt/6.x.x/conanfile.py
 ```
+
+#### [Create SBOM](cmd_create_sbom.py)
+
+Creates an SBOM in CycloneDX format.
+**This command is in an experimental stage, feedback is welcome.**
+
+**Parameters**
+* `sbom_format` _Required_:  `1.4_json`, `1.3_json`, `1.2_json`, `1.4_xml`, `1.3_xml`, `1.2_xml`, `1.1_xml`, or `1.0_xml`
+* supports all other arguments used by `conan graph`, see `conan recipe:create-sbom --help`
+
+**Dependencies**
+The command requires the package `cyclonedx-python-lib`.
+You can install it via
+<!-- keep in sync with the error message that asks for requirements/dependencies in `cmd_create_sbom.py` -->
+```shellSession
+$ pip install 'cyclonedx-python-lib>=3.1.5,<5.0.0'
+```
+
+Usage:
+
+```shellSession
+$ cd conan-center-index/recipes/gmp/all
+$ conan recipe:create-sbom --sbom_format 1.4_json .
```

Comment:
I think we still need to look for a better "layer" name than "recipe"

Reply:
i'd suggest `conan sbom:cyclonedx --format 1.4_xml` as stated here: https://github.com/conan-io/conan-extensions/pull/66#issuecomment-1678740469

Reply:
OK, I changed the name of the command to `conan sbom:cyclonedx --format 1.4_xml`